examples/protocol_buffers/rollup.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  // indicate which modules should be treated as external
+  external: [
```

Comment:
actually globals is the one feature that's still present in the new rule.

It's more ergonomic than having to repeat the things twice here.

BUT maybe we should drop the feature. The attrs should only support parts of the rollup CLI that differ between compilation units. So we could drop unless you can make an argument that a single rollup.config.js would need to be used in different packages where the globals differ.

Reply:
hmmmmm. that's a good question. having it as an attribute means you can take advantage of starlark to populate the attribute.

Reply:
resolved by removing it